components/dash-core-components/src/components/Checklist.react.js

```diff
@@ -187,6 +196,16 @@ Checklist.propTypes = {
      * session: window.sessionStorage, data is cleared once the browser quit.
      */
     persistence_type: PropTypes.oneOf(['local', 'session', 'memory']),
+
+    /**
+     * Indicates whether given `options` are array or dictionary
+     * if True: 
+     *      `options` will receive: [`value1`, `value2`, `value3`, ...]
+     *      which equals to [{label: `value1`, value: `value1`}, {label: `value2, value: `value2`}, ...]
+     * if False: 
+     *      `options` will receive: [{label: `label1`, value: `value1`}, {label: `label2`, value: `value2`}, ...]
+     */
+    inline: PropTypes.bool,
```

Comment:
🔪 Let's remove this and "detect" the shape of `options` within the component itself.